job/dhcp.go

```diff
@@ -139,7 +139,9 @@ func (j Job) setPXEFilename(rep *dhcp4.Packet, isTinkerbellIPXE, isARM, isUEFI,
 	}
 
 	var filename string
+	httpPrefix := conf.PublicFQDN
```

Comment:
This doesn't align with Boots' design philosophy, "_Only func main has the right to decide which flags, env variables, config files are available to the user_" -- [ref](https://github.com/tinkerbell/boots/blob/main/docs/DESIGNPHILOSOPHY.md#:~:text=Only%20func%20main%20has%20the%20right%20to%20decide%20which%20flags%2C%20env%20variables%2C%20config%20files%20are%20available%20to%20the%20user)
Would you be open to passing another value in along with `ipxeBaseURL`?

Reply:
So revert https://github.com/tinkerbell/boots/pull/249/commits/199e821d8c7658eb81b9c61f08fb73b655345562 ?

Reply:
I removed that commit, please take another look.

Reply:
looks good, thank you!